packages/host/webpack.config.js

```diff
@@ -8,104 +9,110 @@ const path = require('path');
+      clean: true,
     },
-  },
-  module: {
-    rules: [
-      {
-        test: /\.[t|j]sx?$/,
-        loader: 'swc-loader',
-        exclude: /node_modules/,
-        options: {
-          jsc: {
-            parser: {
-              dynamicImport: true,
-              syntax: 'typescript',
-              tsx: true,
+    optimization: {
+      splitChunks: {
+        chunks: 'all',
+      },
+    },
+    module: {
+      rules: [
+        {
+          test: /\.[t|j]sx?$/,
+          loader: 'swc-loader',
+          exclude: /node_modules/,
+          options: {
+            jsc: {
+              parser: {
+                dynamicImport: true,
+                syntax: 'typescript',
+                tsx: true,
+              },
+              target: 'es2015',
             },
-            target: 'es2015',
           },
         },
-      },
-    ],
-  },
-  plugins: [
-    new HotModuleReplacementPlugin(),
-
-    new ReactRefreshWebpackPlugin(),
+      ],
+    },
+    plugins: [
+      new HotModuleReplacementPlugin(),
 
-    new BundleAnalyzerPlugin({
-      /**
-       * In "server" mode analyzer will start HTTP server to show bundle report.
-       * In "static" mode single HTML file with bundle report will be generated.
-       * In "json" mode single JSON file with bundle report will be generated
-       */
-      analyzerMode: 'disabled',
-    }),
+      new ReactRefreshWebpackPlugin(),
 
-    new HtmlWebpackPlugin({
-      favicon: './public/favicon.ico',
-      template: './public/index.html',
-    }),
+      new BundleAnalyzerPlugin({
+        /**
+         * In "server" mode analyzer will start HTTP server to show bundle report.
+         * In "static" mode single HTML file with bundle report will be generated.
+         * In "json" mode single JSON file with bundle report will be generated
+         */
+        analyzerMode: 'disabled',
+      }),
 
-    // new ModuleFederationPlugin({
-    //   name: 'host',
-    //   filename: 'remoteEntry.js',
-    //   remotes: {
-    //     customers: 'customers@http://localhost:3007/remoteEntry.js',
-    //     dashboard: 'dashboard@http://localhost:3004/remoteEntry.js',
-    //     transactions: 'transactions@http://localhost:3005/remoteEntry.js',
-    //   },
-    //   exposes: {
-    //     './Host': './src/Host',
-    //   },
-    //   shared: [
-    //     {
-    //       ...deps,
-    //       react: {
-    //         singleton: true,
-    //         requiredVersion: deps.react,
-    //       },
-    //       'react-dom': {
-    //         singleton: true,
-    //         requiredVersion: deps['react-dom'],
-    //       },
-    //     },
-    //     {
-    //       '@openmsupply-client/common': {
-    //         requiredVersion: require('../common/package.json').version,
-    //       },
-    //     },
-    //   ],
-    // }),
-  ],
+      new HtmlWebpackPlugin({
+        favicon: './public/favicon.ico',
+        template: './public/index.html',
+      }),
+      new CopyPlugin({
```

Comment:
tried to use assets, but could not get that to actually copy the file to the output folder. had to use this instead